android/app/src/main/AndroidManifest.xml

```diff
@@ -101,9 +101,7 @@
                 <category android:name="android.intent.category.DEFAULT" />
                 <category android:name="android.intent.category.BROWSABLE" />
 
-                <data android:scheme="content" />
-                <data android:mimeType="*/*" />
```

Comment:
I don't know about pbz, but pbw is a zip archive, so you can use that mime type for it